### PR TITLE
fix: modify build error of point cloud filters in rolling

### PR DESCRIPTION
**Pull request by wep21**
## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: perception/compare_map_segmentation/include/compare_map_segmentation/compare_elevation_map_filter_node.hpp [hunk 1]
```diff
@@ -53,7 +53,7 @@ class CompareElevationMapFilterComponent : public pointcloud_preprocessor::Filte
   void elevationMapCallback(const grid_map_msgs::msg::GridMap::ConstSharedPtr elevation_map);
 
 public:
-  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
+  PCL_MAKE_ALIGNED_OPERATOR_NEW
```
Comment (kenji-miyake) [inline perception/compare_map_segmentation/include/compare_map_segmentation/compare_elevation_map_filter_node.hpp:56]: Could you write some reference links?

File: perception/euclidean_cluster/CMakeLists.txt [hunk 2]
```diff
@@ -9,7 +9,7 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 endif()
 
 find_package(ament_cmake_auto REQUIRED)
-find_package(PCL REQUIRED QUIET COMPONENTS common search filters segmentation)
+find_package(PCL REQUIRED)
```
Comment (kenji-miyake) [inline perception/euclidean_cluster/CMakeLists.txt:12]: Same here.

**Review by kenji-miyake — COMMENTED** [review]
Related: #793, #795

File: perception/compare_map_segmentation/include/compare_map_segmentation/compare_elevation_map_filter_node.hpp (hunk 1 above)
Comment (wep21) [reply perception/compare_map_segmentation/include/compare_map_segmentation/compare_elevation_map_filter_node.hpp:56]: https://github.com/PointCloudLibrary/pcl/blob/3f3038a473a9d97a117822aafe6841b0a9c8612e/common/include/pcl/memory.h#L54-L65

File: perception/euclidean_cluster/CMakeLists.txt (hunk 2 above)
Comment (wep21) [reply perception/euclidean_cluster/CMakeLists.txt:12]: I'm not sure about the reason, but cmake says QHULL::QHULL and libusb::libusb could not be found. I will investigate the reasons.

Comment (kenji-miyake) [reply perception/euclidean_cluster/CMakeLists.txt:12]: Hmm? How are `QHULL` and `libusb` related to this line? :thinking:

**Review by kenji-miyake — APPROVED** [review]

File: perception/euclidean_cluster/CMakeLists.txt (hunk 2 above)
Comment (kenji-miyake) [reply perception/euclidean_cluster/CMakeLists.txt:12]: Ah, probably other PCL components have dependencies to `QHULL` and `libusb`, and removing components imports all packages, so it's resolved, right?

Comment (wep21) [reply perception/euclidean_cluster/CMakeLists.txt:12]: Yes, I guess so.

**Pull request merged**